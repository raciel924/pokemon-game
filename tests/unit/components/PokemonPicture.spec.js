import {shallowMount} from '@vue/test-utils'
import pokemonPicture from '@/components/PokemonPicture'

describe('pokemonPicture componente', ( ) => {
    test('debe de hacer el match con el snapshot', () => {
        
    })
    test('debe de mostrar la imagen oculta y el pokemon 100', () => {
        const wrapper = shallowMount(pokemonPicture,{
            props: {
                pokemonID:1,
                showPokemon:false,
            }
        }) 
       const {img1, img2} = wrapper.findAll('div')
       console.log(img1)
       // expect(img1.toExist()).toBeTruthy()
       // expect(img2).toBeTruthy(undefined)
       // expect(img1.classes('hiden-pokemon')).toBe(true)
    })
    test('debe de mostrar el pokemon si showPokemon:true', () => {
        
    })
})