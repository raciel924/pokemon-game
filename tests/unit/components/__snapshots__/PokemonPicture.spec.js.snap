// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pokemonPicture componente debe de hacer el match con el snapshot 1`] = `<div class="pokemon-container" pokemon="1"><img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/undefined.svg" class="hiden-pokemon" alt="pokemon"></div>`;
